### ISSUE TITLE
WordPress source

### ISSUE THREAD

**Issue by marekxhtml** [xfiveco/generator-xh#209]
I think that WordPress files should be downloaded from the https://wordpress.org/latest.zip or https://wordpress.org/latest.tar.gz rather than github. There is a beta version on the master branch, so if we would like to still use github for the generator we should change a branch to the 4.4 version.
I think that the best way will be just use the latest.zip link, because it is always the latest stable version.

**Comment by luboskmetko** [comment]
In some early versions there was a `simple-git` function for finding out the latest tags in the WP repository, but it was then removed for some reason. I couldn't make this work again, so for now I have hardcoded 4.4-branch into the wp generator https://github.com/xfiveco/generator-xh/blob/master/wp/index.js#L101. This should fetch the latest stable code.

Probably it would be better to fetch and extract the zip as you suggested but there doesn't seem to be a quick way to do that in node.
